portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,38 @@
+.navigation {
+    background-color: black;
+} 
+
+.navigation a {
+    display: inline-block;
+    text-align: center;
+    color: white;
+    padding: 20px;   
+    text-decoration: none; 
+} 
+
+.navigation a:hover {
+    background-color: white;
+    color: black;  
+} 
+
+h1 {
+    font-size: 100px;
+    color: white;
+    padding-top: 150px;
+}
+
+html {
+    background: url(images/try.gif);
+    background-size: cover;
+    background-repeat: no-repeat;
+}
```

Comment:
Personal preference/Optional Suggestion: I like to have the more broad selectors at the start of the file.